test/hotspot/jtreg/compiler/c2/TestCastIIWrongTypeCCP.java

```diff
@@ -0,0 +1,63 @@
+/*
```

Comment:
This file might fit better under `test/hotspot/jtreg/compiler/ccp`.

Reply:
That's a good point, I'll move it over there.